src/IntercomContacts.php

```diff
@@ -78,6 +84,38 @@ public function deleteContact(string $id, array $options = [])
         return $this->client->delete($path, $options);
     }
 
+    /**
+     * Applys a tag to a Contact based on the provided Tag ID
+     *
+     * @see    https://developers.intercom.com/intercom-api-reference/reference#tag-contact
+     * @param  string $id
+     * @param  string  $tag_id
```

Comment:
We should use camelCase for all variables in this repo 🙌 

```suggestion
     * @param  string $tagId
```

Reply:
WeCanCertainlyDoThat.

Fixed.